### PR TITLE
Naze32: Add S.BUS inverter support + fix DSM bind->inverter conflict

### LINKED ISSUE #577: Naze: Support UART inverter (rev. 6)
Pins https://github.com/multiwii/baseflight/blob/master/src/board.h#L216.

### PR DESCRIPTION
**Pull request by tracernz**
Completely untested because I don't have any inverter/inverted hardware at all.

Fixes #577

### CODE REVIEW

**Comment by mlyle** [comment]
@fujin do you have nazeR6 + sbus hw?

**Comment by fujin** [comment]
yep, I can put the radio I was using on my lux on an r6 and test it.

**Comment by fujin** [comment]
jenkins, build artifacts please

File: flight/targets/naze32/fw/pios_board.c [hunk 1]
```diff
@@ -424,6 +335,110 @@ void PIOS_Board_Init(void) {
...
+				NULL);                                  // sbus_cfg
+		break;
+
+	case HWNAZE_RCVRPORT_PPMSERIAL:
+	case HWNAZE_RCVRPORT_SERIAL:
+		{
+			uint8_t hw_rcvrserial;
+			HwNazeRcvrSerialGet(&hw_rcvrserial);
+
+			HwNazeDSMxModeOptions hw_dsmx_mode;
```
Comment (mlyle) [inline flight/targets/naze32/fw/pios_board.c:366]: This switch is something other targets are missing.  We should figure out how to factor this into PIOS_HAL.

**Comment by fujin** [comment]
Confirmed fixed.

image

image

Sorry for the delay. LGTM :+1:

**Pull request merged**